samples/Shared/UnitTestFramework/CodeFixVerifier`2.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Roslyn.UnitTestFramework
+{
+    public static class CodeFixVerifier<TAnalyzer, TCodeFix>
+        where TAnalyzer : DiagnosticAnalyzer, new()
+        where TCodeFix : CodeFixProvider, new()
+    {
+        public static DiagnosticResult[] EmptyDiagnosticResults
+            => DiagnosticVerifier<TAnalyzer>.EmptyDiagnosticResults;
+
+        public static DiagnosticResult Diagnostic(string diagnosticId = null)
+            => DiagnosticVerifier<TAnalyzer>.Diagnostic(diagnosticId);
+
+        public static DiagnosticResult Diagnostic(DiagnosticDescriptor descriptor)
+            => DiagnosticVerifier<TAnalyzer>.Diagnostic(descriptor);
+
+        public static DiagnosticResult CompilerError(string errorIdentifier)
+            => DiagnosticVerifier<TAnalyzer>.CompilerError(errorIdentifier);
+
+        public static Task VerifyCSharpDiagnosticAsync(string source, DiagnosticResult expected, CancellationToken cancellationToken)
```

Comment:
Remove the cancellation token or make it optional? Only keep it if there is a use case for testing cancellation.

Reply:
➡️ Should be fixed now 👍